design-reviews/recipes/milestone-epic-design-review-recipe.md

```diff
@@ -0,0 +1,79 @@
+# Milestone / Epic Design Review Recipe
+
+## Things to keep in mind
+
+* As with all other aspects of the project, design reviews must provide a friendly and safe environment so that any team member feels comfortable proposing a design for review and can use the opportunity to grow and learn from the constructive / non-judgemental feedback from peers and subject matter experts (see [Team Agreements](../../team-agreements)).
+* Design reviews should be lightweight and should not feel like an additional process overhead.
+* Tech Lead can usually provide guidance on whether a given epic/milestone needs a design review and can help other team members in preparation.
+* This is *not* a strict template that must be followed and teams should not be bogged down with polished "design presentations".
+* Think of the recipe below as a "menu of options" for potential questions to think through in designing this epic. Not all sections are required for every epic. Focus on sections and questions that are most relevant for making the decision and rationalizing the trade offs.
+* Milestone/epic design is considered high-level design but is usually more detailed than the design included in the Game Plan, but will likely re-use some of the technologies, non-functional requirements, and constraints mentioned in the Game Plane.
+* As the team learned more about the project and further refined the scope of the epic, they may specifically call out notable changes to the overall approach and, in particular, highlight any unique deployment, security, private, scalability, etc. characteristics of this milestone.
+
+***
+
+# [Design Title]
+
+* Milestone / Epic: [Name](http://link-to-work-item)
+* Project / Engagement: [Project Engagement]
+* Authors: [Authors]
+
+## Overview / Problem Statement
+
+> Describe the milestone/epic with a high-level summary and a problem statement. Consider including or linking to any additional background (e.g. Game Plan or Checkpoint docs) if it is useful for historical context.
+
+## Goals / In-Scope
+
+> List bullet points of goals that the milestone/epic will help achieve. Include acceptable criteria required to meet the [Definition of Done](../../team-agreements/definition-of-done/).
+
+## Non-goals / Out-of-Scope
+
+> List bullet points of non-goals for this milestone/epic to clarify the work that is beyond the scope of this milestone/epic.
+
+## Proposed Design / Suggested Approach
+

```

Comment:
I would add guidance that this should be extremely brief and short. Most likely, this will change quite significantly once the epic is further decomposed. Therefore, i would try to limit the time investment. I would suggest one logical architecture diagram, with a 1 paragraph explainer. The goal being to convey the vision and complexity in something that can be digested in a few minutes.

Reply:
Agree and will make this explicit.